velox/functions/prestosql/aggregates/AggregateNames.h

```diff
@@ -51,5 +51,5 @@ const char* const kSum = "sum";
 const char* const kVariance = "variance"; // Alias for var_samp.
 const char* const kVarPop = "var_pop";
 const char* const kVarSamp = "var_samp";
-
+const char* const kMaxSizeForStats = "$internal$max_data_size_for_stats";
```

Comment:
In Presto, this function was an internal function not exposed to users. However, in Velox, this becomes a publicly visible function that can be used by applications such as Presto. Hence, $internal$ in the name no longer makes sense. Perhaps, drop $internal$ prefix.

Reply:
@mbasmanova 

I believe that that may not work because this name is plumbed through from coordinator.

Reply:
Prestissimo has logic to translate functions. This logic can be modified to accommodate the name change.

See mapScalarFunction and mapAggregateFunctionName in presto_cpp/main/types/PrestoToVeloxExpr.cpp

Reply:
Given that this involves Prestissimo and will simply be a string change, I recommend we do that in a separate MR?

Reply:
It would be a bit strange to introduce a function with the "wrong" name and then follow-up with a name change.

Reply:
fixed.